src/runtime/composables/useClientStripe.ts

```diff
@@ -22,11 +22,17 @@ export default async function useClientStripe() {
     isLoading.value = true
 
     if (!nuxtApp.$config.public.stripe.key) console.warn("no key given for Stripe client service")
-
-    return await loadStripe(
-      nuxtApp.$config.public.stripe.key,
-      nuxtApp.$config.public.stripe.options
```

Comment:
with your `function useClientStripe(clientOptions?: StripeConstructorOptions)`
maybe:
```
      return await loadStripe(
          nuxtApp.$config.public.stripe.key
          options: clientOptions ?? nuxtApp.$config.public.stripe.options
      )
```

Reply:
Sorry for replying a bit late.

This makes the code simpler i would try this in the end of the week
Best regards

Reply:
@Thomashh2k did you try?